addOns/spiderAjax/src/main/javahelp/org/zaproxy/zap/extension/spiderAjax/resources/help/contents/automation.html

```diff
@@ -29,6 +27,14 @@ <H2>Job: spiderAjax</H2>
       maxCrawlStates:                  # Int: The maximum number of crawl states the crawler should crawl, default: 0 unlimited
       randomInputs:                    # Bool: When enabled random values will be entered into input element, default: true
       reloadWait:                      # Int: The time in millseconds to wait after the URL is loaded, default: 1000
+    tests:
```

Comment:
I think it would be worth adding a generic tests help page which would explain that you can add as many tests as you like to any jobs. We could then link from there to more info about the stats, if we create that sometime :P